src/views/admin/KlasseDetailsView.vue

```diff
@@ -226,20 +244,40 @@
               v-if="!isEditActive"
               class="d-flex justify-sm-end"
             >
-              <v-col
-                cols="12"
-                sm="6"
-                md="auto"
-              >
-                <v-btn
-                  class="primary ml-lg-8"
-                  data-testid="klasse-edit-button"
-                  @Click="activateEditing"
-                  :block="mdAndDown"
+              <v-row class="pt-3 px-2 justify-end">
+                <v-col
+                  cols="12"
+                  md="auto"
+                  sm="6"
+                >
+                  <div class="d-flex justify-sm-end">
+                    <KlasseDelete
+                      :errorCode="organisationStore.errorCode"
+                      :klassenname="organisationStore.currentKlasse?.name"
+                      :klassenId="organisationStore.currentKlasse?.id"
+                      ref="klasse-delete"
+                      :schulname="selectedSchule"
+                      :useIconActivator="false"
+                      @onDeleteKlasse="deleteKlasseById(currentOrganisationId)"
+                    >
+                    </KlasseDelete>
+                  </div>
+                </v-col>
+                <v-col
+                  cols="12"
+                  sm="6"
+                  md="auto"
                 >
-                  {{ $t('edit') }}
-                </v-btn>
-              </v-col>
+                  <v-btn
+                    class="primary ml-lg-8"
+                    data-testid="klasse-edit-button"
+                    @Click="activateEditing"
```

Comment:
```suggestion
                    @click="activateEditing"
```
Oder ist das eine notation, die ich nicht kenne?

Reply:
@click = @Click eigentlich das gleiche, habe aber trotzdem geändert